google-cloud-spanner/src/main/java/com/google/cloud/spanner/BatchTransactionId.java

```diff
@@ -32,12 +32,18 @@ public class BatchTransactionId implements Serializable {
   private final ByteString transactionId;
   private final String sessionId;
   private final Timestamp timestamp;
+  private final boolean isMultiplexedSession;
```

Comment:
Do we need this flag for anything? Otherwise, I think we should remove it. See also my comment above; If a client receives a session ID from another client, then this client does not know whether that session ID is for a regular or a multiplexed session. If such an ID is then used to create a `BatchTransactionId`, then this could lead to a `BatchTransactionId` that contains 'false' information.